index.js

```diff
@@ -27,15 +29,22 @@ module.exports = ({
   hfDSBinancePort = 23522,
   hfDSBitfinexPort = 23521
 }) => {
-  const dbBitfinex = new HFDB({
-    schema: HFDBBitfinexSchema,
-    adapter: HFDBLowDBAdapter({ dbPath: hfBitfinexDBPath })
-  })
+  let dbBitfinex = null
+  let dbBinance = null
 
-  const dbBinance = new HFDB({
-    schema: HFDBBinanceSchema,
-    adapter: HFDBLowDBAdapter({ dbPath: hfBinanceDBPath })
-  })
+  if (hfBitfinexDBPath && hfDSBitfinexPort) {
```

Comment:
maybe drop a debug log in each of these code paths to show the service is connecting to a new DB

Reply:
might not be needed actually since the dataserver spits out some logs, ill leave it up to you